src/scripts.js

```diff
@@ -50,24 +79,18 @@ function startApp() {
 }
 
 function addInfoToSidebar(user, userStorage) {
-  $("#sidebarName").text(user.name);
-  $("#headerText").text(`${user.getFirstName()}'s Activity Tracker`);
-  $("#stepGoalCard").text(`Your daily step goal is ${user.dailyStepGoal}.`);
-  $("#userAddress").text(user.address);
-  $("#userEmail").text(user.email);
-  $("#userStridelength").text(
-    `Your stridelength is ${user.strideLength} meters.`
-  );
-  $("#friendList").append(makeFriendHTML(user, userStorage));
-}
+  sidebarName.innerText = user.name;
```

Comment:
We'll have to fix this tomorrow, but the jQuery selectors were correct. (Not sure how this ended up like this, though)